webpack/webpack.dev.config.js

```diff
@@ -63,10 +63,7 @@ module.exports = Merge.smart(commonConfig, {
         issuer: {
           test: /\.jsx?$/,
         },
-        exclude: [
```

Comment:
This exclusion wasn't necessary?

Reply:
It didn't seem so. I figured I should remove it if it wasn't having an impact anymore. It could be that this was added to fix the extra loader problem that the certificate images were also having.

Reply:
Guess we'll see on staging. :)